### PR TITLE
LPS-122881 Make CalendarPortletDataHandler a display portlet.

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by rafaprax** [comment]
Hey @MarinhoFeliphe, related error here.

`com.liferay.calendar.internal.exportimport.data.handler.test.CalendarPortletDataHandlerTest.testIsDisplayPortlet`

**Comment by MarinhoFeliphe** [comment]
Thanks, @rafaprax 

I can't override this method that's way, I am worked on it.

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Hey @MarinhoFeliphe, the 901717d commit is not required for your fix. Can you please remove it and forward this pr? Thx

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:stop

**Comment by rafaprax** [comment]
ci:help

**Comment by rafaprax** [comment]
ci:stop

**Comment by rafaprax** [comment]
ci:test:calendar

**Comment by rafaprax** [comment]
ci:stop

**Comment by rafaprax** [comment]
ci:stop

**Comment by rafaprax** [comment]
ci:

**Comment by rafaprax** [comment]
ci:forward

File: modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/exportimport/data/handler/CalendarPortletDataHandler.java [hunk 1]
```diff
@@ -28,15 +35,70 @@
...
-	protected void activate() {
-		super.activate();
+	public String getSchemaVersion() {
+		return SCHEMA_VERSION;
+	}
 
+	@Activate
+	protected void activate() {
 		setDataLevel(DataLevel.PORTLET_INSTANCE);
+		setDataPortletPreferences(
```
Comment (rafaprax) [inline modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/exportimport/data/handler/CalendarPortletDataHandler.java:50]: Hey @MarinhoFeliphe , sort the portletPreferences name alphabetically, please. Apply this for other places as well.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — COMMENTED** [review]
Thanks, @rafaprax .

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**